tests/unit/test_webhook_model.py

```diff
@@ -139,7 +139,7 @@ def test_comment_closed_reason(mock_mr):
 @patch('webcompat.webhooks.model.make_request')
 def test_moderate_public_issue(mock_mr):
     """Test issue state and API request that is sent to GitHub."""
-    mock_mr.return_value.status_code == 200
+    mock_mr.return_value.status_code = 200
     json_event, signature = event_data('private_issue_opened.json')
```

Comment:
not related to this work, but I think we should be setting a status with `= 200`? as right not it's not doing anything

Reply:
Thanks for catching this. Yes definitely right.